config/software/tilting/qgl_180mm.cfg

```diff
@@ -0,0 +1,28 @@
+#for micron+
+
+# Quand gantry leveling definition
+# If this files is included, then it also activate the QGL
+# automatically in the START_PRINT macro
+[gcode_macro _USER_VARIABLES]
+variable_qgl_enabled: True
+gcode:
+
+# Also include directly the dockable probe overide of qgl from here
+[include ../../../macros/base/probing/overides/qgl.cfg]
+[include ../../../macros/base/homing/tilting.cfg]
+
+
+[quad_gantry_level]
+gantry_corners:
+   -60.2,-10.4
+   244.1,234.5
+points:
+   20,30
+   20,180
+   170,180
+   170,30
```

Comment:
Same question about the points here: 180 seems high for a 180mm bed. Is this right?